### PR TITLE
To handle node driver changes in gke pipeline

### PR DESCRIPTION
**Pull request by sn-px**
**What this PR does / why we need it** :
To incorporate the node driver changes 

**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PB-5875

**Special notes for your reviewer**:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/view/Px-Backup%20Nightly%20Run/job/basic-system-test/job/px/job/s3/job/PX-backup-px-gke/85/console

### CODE REVIEW

File: deployments/deploy-pxb-cloud.sh [hunk 1]
```diff
@@ -356,9 +356,6 @@ if [ -z "${NODE_DRIVER}" ]; then
 fi
 if [ -n "${K8S_VENDOR}" ]; then
     case "$K8S_VENDOR" in
-        gke)
-            NODE_DRIVER="gke"
-            ;;
```
Comment (mkoppal-px) [inline deployments/deploy-pxb-cloud.sh:361]: So node driver will be SSH for this?

File: deployments/deploy-pxb-cloud.sh [hunk 2]
```diff
@@ -367,7 +364,6 @@ if [ -n "${K8S_VENDOR}" ]; then
             ;;
     esac
 fi
-
```
Comment (mkoppal-px) [inline deployments/deploy-pxb-cloud.sh:370]: Remove this diff

File: deployments/deploy-pxb-cloud.sh (hunk 1 above)
Comment (sn-px) [reply deployments/deploy-pxb-cloud.sh:361]: yes

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**